### PR TITLE
dump git server logs on test failure

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
[testextended][extended:core(can use private repositories as build input)]

**Comment by bparees** [comment]
[merge]

File: test/extended/builds/gitauth.go [hunk 1]
```diff
@@ -80,6 +80,9 @@ var _ = g.Describe("[builds][Slow] can use private repositories as build input",
 
 		g.By("starting a test build")
```
Comment (black1977) [inline test/extended/builds/gitauth.go]: /start

Comment (bparees) [inline test/extended/builds/gitauth.go]: starting is correct in the syntax of gomega.  "by [starting a test build]"

**Comment by bparees** [comment]
flake https://github.com/openshift/origin/issues/13105
[merge]

**Pull request merged**